posts/2020-12-15-rust-survey-2020.md

```diff
@@ -38,7 +38,7 @@ Here's the distribution of languages across the responses:
 
 While Rust itself has always had a strong stability guarantee, stability means more than just ensuring users’ code doesn’t break with a new version of the compiler. Rust in 2020 has largely been about cleaning up and stabilizing features and initiatives that were already under way. While this work is not nearly completed, respondents have noted that the stability of Rust in general has been improving.
 
-First, we’d like to make a shout out to the [rust-analyzer] and [Intellij Rust plugin][rust-intellij] projects which both enjoy relatively happy user bases. Nearly 3/4ths of all respondents noted that they saw at least some improvement in the IDE story, but users of rust-analyzer and Intellij were especially happy with 47% of rust-analyzer users noting “a lot of improvement” while 40% of Intellij users said the same. 
+First, we’d like to make a shout out to the [rust-analyzer] and [InteliJ Rust plugin][rust-intellij] projects which both enjoy relatively happy user bases. Nearly 3/4ths of all respondents noted that they saw at least some improvement in the IDE story, but users of rust-analyzer and InteliJ were especially happy with 47% of rust-analyzer users noting “a lot of improvement” while 40% of InteliJ users said the same. 
```

Comment:
I think this should stay `intellij`, with two l's.
```suggestion
First, we’d like to make a shout out to the [rust-analyzer] and [IntelliJ Rust plugin][rust-intellij] projects which both enjoy relatively happy user bases. Nearly 3/4ths of all respondents noted that they saw at least some improvement in the IDE story, but users of rust-analyzer and IntelliJ were especially happy with 47% of rust-analyzer users noting “a lot of improvement” while 40% of IntelliJ users said the same. 
```

Reply:
You're right. I misread. 😆